src/libs/PersonalDetailsUtils.ts

```diff
@@ -195,6 +201,66 @@ function getEffectiveDisplayName(personalDetail?: PersonalDetails): string | und
     return undefined;
 }
 
+/**
+ * Creates a new displayName for a user based on passed personal details or login.
+ *
+ * @param login - user's login
+ * @param passedPersonalDetails - details object with firstName and lastName
+ * @returns - The effective display name
```

Comment:
This is not needed in TS if params are self-explanatory
```suggestion
```

Reply:
@situchan but other methods in this file also have params listed

Reply:
https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#jsdoc

<img width="1219" alt="Screenshot 2024-01-29 at 2 59 53 PM" src="https://github.com/Expensify/App/assets/108292595/e8ccac17-d96e-4819-bcb1-39b1038850ae">

Reply:
@situchan should I remove the params form other methods' docs as well? wdyt?

Reply:
not a big deal

Reply:
Let’s remove it from this method though, yeah?